test/e2e/set/scyllacluster/scyllacluster_updates.go

```diff
@@ -0,0 +1,150 @@
+
+		sc, err = f.ScyllaClient().ScyllaV1().ScyllaClusters(f.Namespace()).Create(ctx, sc, metav1.CreateOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Waiting for the ScyllaCluster to deploy")
+		waitCtx1, waitCtx1Cancel := contextForRollout(ctx, sc)
+		defer waitCtx1Cancel()
+		sc, err = waitForScyllaClusterState(waitCtx1, f.ScyllaClient().ScyllaV1(), sc.Namespace, sc.Name, scyllaClusterRolledOut)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		di, err := NewDataInserter(ctx, f.KubeClient().CoreV1(), sc, getMemberCount(sc))
+		o.Expect(err).NotTo(o.HaveOccurred())
+		defer di.Close()
+
+		err = di.Insert()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		verifyScyllaCluster(ctx, f.KubeClient(), sc, di)
+
+		framework.By("Changing pod resources")
+		oldResources := *sc.Spec.Datacenter.Racks[0].Resources.DeepCopy()
+		newResources := *oldResources.DeepCopy()
+		o.Expect(oldResources.Requests).To(o.HaveKey(corev1.ResourceCPU))
+		o.Expect(oldResources.Requests).To(o.HaveKey(corev1.ResourceMemory))
+		o.Expect(oldResources.Limits).To(o.HaveKey(corev1.ResourceCPU))
+		o.Expect(oldResources.Limits).To(o.HaveKey(corev1.ResourceMemory))
+
+		newResources.Requests[corev1.ResourceCPU] = *addQuantity(newResources.Requests[corev1.ResourceCPU], resource.MustParse("1m"))
+		newResources.Requests[corev1.ResourceMemory] = *addQuantity(newResources.Requests[corev1.ResourceMemory], resource.MustParse("1Mi"))
+		o.Expect(newResources.Requests).NotTo(o.BeEquivalentTo(oldResources.Requests))
+
+		newResources.Limits[corev1.ResourceCPU] = *addQuantity(newResources.Limits[corev1.ResourceCPU], resource.MustParse("1m"))
+		newResources.Limits[corev1.ResourceMemory] = *addQuantity(newResources.Limits[corev1.ResourceMemory], resource.MustParse("1Mi"))
+		o.Expect(newResources.Limits).NotTo(o.BeEquivalentTo(oldResources.Limits))
+
+		o.Expect(newResources).NotTo(o.BeEquivalentTo(oldResources))
+
+		newResourcesJSON, err := json.Marshal(newResources)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		sc, err = f.ScyllaClient().ScyllaV1().ScyllaClusters(f.Namespace()).Patch(
+			ctx,
+			sc.Name,
+			types.JSONPatchType,
+			[]byte(fmt.Sprintf(
+				`[{"op": "replace", "path": "/spec/datacenter/racks/0/resources", "value": %s}]`,
+				newResourcesJSON,
+			)),
+			metav1.PatchOptions{},
+		)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(sc.Spec.Datacenter.Racks[0].Resources).To(o.BeEquivalentTo(newResources))
+
+		framework.By("Waiting for the ScyllaCluster to redeploy")
+		waitCtx2, waitCtx2Cancel := contextForRollout(ctx, sc)
+		defer waitCtx2Cancel()
+		sc, err = waitForScyllaClusterState(waitCtx2, f.ScyllaClient().ScyllaV1(), sc.Namespace, sc.Name, scyllaClusterRolledOut)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Changing requested storage capacity")
+		// Requested capacity is only manifested on new PVC the old ones have to be updated manually because
+		// StatefulSets are not reconciling PVCs, only create it initially.
```

Comment:
Lets verify that old member pvc has previous storage capacity, and after scaling, new member has new storage capacity

Reply:
actually statefulset forbid the update so we'd have to orphan delete them and recreate which isn't worth the effort now I guess